README.md

```diff
@@ -113,6 +113,27 @@ fplll can also be built from source. Below, we explicate some of the dependencie
   
 NOTE: If you are intending to use fplll on Windows 10, then these packages should be installed after the `Windows Subsystem for Linux` has been installed and activated. Please go to the [Windows 10](#windows-10) instructions for more information.
 
+### Using `cmake` instead of `libtools` and `auto[conf|make]` ###
+
+_Note:_ the `CMakeLists.txt` configuration is unofficial and trying to reproduce the official build as close as possible, however, less tested cross-platforms.
```

Comment:
I think if we're going to add this, we should add it to the CI. Otherwise, there's no guarantee that this won't drift from the regular autotools build files. I'm happy to help with this if you'd like.

Reply:
sounds good! and would definitely appreciate any help from you!